doc/sphinx/source/getting_started/running.rst

```diff
@@ -4,17 +4,96 @@
 Running ESMValTool
 ******************
 
-To run ESMValTool, use the command
+ESMValTool is mostly used as a command line tool. Whenever your
+conda environment for ESMValTool is active, you can just run the command
+``esmvaltool <options> <recipe>``. One of the options that must be specified
+is the user-config file, which is specified using the
+option ``-c /path/to/config-user.yaml``. An example recipe is available in the
+ESMValTool installation folder as ``examples/recipe_python.yml``:
```

Comment:
Make this a link to https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/recipes/examples/recipe_python.yml